Mathlib/Data/Nat/Cast/Order/Basic.lean

```diff
@@ -69,6 +69,9 @@ variable [CharZero α] {m n : ℕ}
 theorem strictMono_cast : StrictMono (Nat.cast : ℕ → α) :=
   mono_cast.strictMono_of_injective cast_injective
 
+@[gcongr]
+lemma _root_.GCongr.natCast_lt_natCast {a b : ℕ} (h : a < b) : (a : α) < b := strictMono_cast h
```

Comment:
Can we just make this an `alias`? And add a docstring to that effect?

Reply:
An alias of what? `strictMono_cast`? `gcongr` doesn't understand `StrictMono`